cob/cli/generate_cli.py

```diff
@@ -146,4 +146,4 @@ def generate(self, dest_path):
         if os.path.exists(dest_path):
             raise click.ClickException("{} already exists".format(normalized_path))
         with open(dest_path, 'w') as f:
-            f.write(template.render(**_ctx))
+            f.write(template.render(**_ctx)+'\n')
```

Comment:
I think it's better to ensure all templates are newline-terminated. Some files generated might be sensitive to terminating newlines...

Reply:
You are right, sorry.
I pushed a new commit instead of this one.
Apparently, `jinja2.Environment` class had removed the new lines which were in the templates - I passed `keep_trailing_newline=True` to fix it.